src/pages/komodo-defi-framework/api/v20/start_simple_market_maker_bot/index.mdx

```diff
@@ -13,7 +13,7 @@ For convenience, an online [tool for generating configs](https://stats.kmd.io/at
 
 | Structure                                                | Type    | Description                                                                                                                                                                                                 |
 | -------------------------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| price\_url                                               | string  | Link to a price service API                                                                                                                                                                                 |
+| price\_url                                               | list    | A list of URLs serving price data in JSON format. During each update loop, price data will be sourced from the first working URL in the list.                                                               |
```

Comment:
if it's a list, the parameter should be called `price_urls` https://github.com/KomodoPlatform/komodo-defi-framework/pull/2027#discussion_r1447080000. We kept the `price_url` string parameter for backward compatibility, so maybe we should document `price_urls` only and not both, it's up to you. This is optional btw, and it will use a default list if not specified.